packages/connect/src/core/index.ts

```diff
@@ -231,13 +231,17 @@ const initDevice = async (method: AbstractMethod<any>) => {
 
     if (method.devicePath) {
         device = _deviceList.getDevice(method.devicePath);
-        showDeviceSelection = !device || !!device?.unreadableError;
+        showDeviceSelection =
+            !device ||
+            !!device?.unreadableError ||
+            // Suite calls getFeatures on unacquired device to acquire it
+            (device.isUnacquired() && method.name != 'getFeatures');
```

Comment:
hmm, it makes sense actually but I don't like this line very much. 

the only other option would could be doing automated acquire in this case 🤔

Reply:
To solve this more cleanly we would probably have to add a new method/parameter that Suite can call to acquire the device
https://github.com/trezor/trezor-suite/blob/185a936bc0cf12cffa30ac4ff5c1629da44d6178/suite-common/wallet-core/src/device/deviceThunks.ts#L253-L278

Reply:
maybe lets check if using popup